recipes/singlem/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "singlem" %}
+{% set version = "0.13.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ba1f629ae89b05d0fd87e978bf0094dad5bac1bece56fb1d700f1987fb424cd8
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - nose >=1.0
```

Comment:
nose can probably be removed from the host requirements

Reply:
I can't get it to build without it, unfortunately.